packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -281,10 +284,8 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   }
 
   void _handleConnectionStart(VmService service) async {
```

Comment:
this method seems like it doesn't need to be async anymore. probably the cleanest solution is to remove the async and leave the setState where it was.